### DISCUSSION TITLE
Password reset flow when using phone number and password to authenticate

### DISCUSSION THREAD

**Discussion by OyugoObonyo** [supabase/supabase#13390] [Questions]
Can I implement a password reset when a user has logged in with just a phone number and password? The current reset password API sends the password reset link to a user's email yet our application isn't collecting user emails. How can I achieve a password reset functionality with just a phone number? Also, can I invite users by magic link my application through their phone numbers or is logging in by magic link restricted to just emails?

**Comment by akeva001** [comment]
Would love this as well!

**Comment by sumersao** [comment]
This would be such a good feature. I'd like it as well!
